assert/assertions.go

```diff
@@ -75,6 +89,47 @@ func ObjectsAreEqual(expected, actual interface{}) bool {
 	return bytes.Equal(exp, act)
 }
 
+// DEVOTED PATCH
+// GetInternalProtobufFields returns the names of private fields that may be injected by proto-c-gen-go.
+func GetInternalProtobufFields() []string {
+	return []string{"state", "sizeCache", "unknownFields", "XXX_unrecognized", "XXX_sizecache", "XXX_NoUnkeyedLiteral"}
+}
```

Comment:
Could probably do it later (and perhaps it will make a tiny difference), but a  package variable map would be probably more efficient and could get rid of `InStringArray`